<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<title>"prepare ATAC-seq data from Luca/Pique-Regi lab for analysis"</title>
<!-- 2016-11-01 Tue 12:39 -->
<meta  http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta  name="generator" content="Org-mode" />
<meta  name="author" content="Heejung Shim" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center; }
  .todo   { font-family: monospace; color: red; }
  .done   { color: green; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: visible;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline;}
  pre.src-sh:before    { content: 'sh'; }
  pre.src-bash:before  { content: 'sh'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-R:before     { content: 'R'; }
  pre.src-perl:before  { content: 'Perl'; }
  pre.src-java:before  { content: 'Java'; }
  pre.src-sql:before   { content: 'SQL'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.right  { text-align: center;  }
  th.left   { text-align: center;   }
  th.center { text-align: center; }
  td.right  { text-align: right;  }
  td.left   { text-align: left;   }
  td.center { text-align: center; }
  dt { font-weight: bold; }
  .footpara:nth-child(2) { display: inline; }
  .footpara { display: block; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  /*]]>*/-->
</style>
<script type="text/javascript">
/*
@licstart  The following is the entire license notice for the
JavaScript code in this tag.

Copyright (C) 2012-2013 Free Software Foundation, Inc.

The JavaScript code in this tag is free software: you can
redistribute it and/or modify it under the terms of the GNU
General Public License (GNU GPL) as published by the Free Software
Foundation, either version 3 of the License, or (at your option)
any later version.  The code is distributed WITHOUT ANY WARRANTY;
without even the implied warranty of MERCHANTABILITY or FITNESS
FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.

As additional permission under GNU GPL version 3 section 7, you
may distribute non-source (e.g., minimized or compacted) forms of
that code without the copy of the GNU GPL normally required by
section 4, provided you include this license notice and a URL
through which recipients can access the Corresponding Source.


@licend  The above is the entire license notice
for the JavaScript code in this tag.
*/
<!--/*--><![CDATA[/*><!--*/
 function CodeHighlightOn(elem, id)
 {
   var target = document.getElementById(id);
   if(null != target) {
     elem.cacheClassElem = elem.className;
     elem.cacheClassTarget = target.className;
     target.className = "code-highlighted";
     elem.className   = "code-highlighted";
   }
 }
 function CodeHighlightOff(elem, id)
 {
   var target = document.getElementById(id);
   if(elem.cacheClassElem)
     elem.className = elem.cacheClassElem;
   if(elem.cacheClassTarget)
     target.className = elem.cacheClassTarget;
 }
/*]]>*///-->
</script>
</head>
<body>
<div id="content">
<h1 class="title">"prepare ATAC-seq data from Luca/Pique-Regi lab for analysis"</h1>
<div id="table-of-contents">
<h2>Table of Contents</h2>
<div id="text-table-of-contents">
<ul>
<li><a href="#sec-1">1. Copy new ATAC-seq data from Luca/Pique-Regi lab to the rice cluster</a></li>
<li><a href="#sec-2">2. Convert bam file to hdf5 format</a>
<ul>
<li><a href="#sec-2-1">2.1. Prepare directory and install genome library, samtools, and python</a>
<ul>
<li><a href="#sec-2-1-1">2.1.1. Create directory</a></li>
<li><a href="#sec-2-1-2">2.1.2. Install genome library</a></li>
<li><a href="#sec-2-1-3">2.1.3. Install samtools</a></li>
<li><a href="#sec-2-1-4">2.1.4. Install python</a></li>
</ul>
</li>
<li><a href="#sec-2-2">2.2. Copy chromosome.h5 file to rice cluster</a></li>
<li><a href="#sec-2-3">2.3. Filter reads with low alignment quality 10</a>
<ul>
<li><a href="#sec-2-3-1">2.3.1. Create directory to save preprocessed bam files</a></li>
<li><a href="#sec-2-3-2">2.3.2. Filter reads with low alignment quality 10</a></li>
</ul>
</li>
<li><a href="#sec-2-4">2.4. Index filtered bam files</a></li>
<li><a href="#sec-2-5">2.5. Convert filtered bam files into hdf5</a></li>
<li><a href="#sec-2-6">2.6. Obtain library read depth</a></li>
</ul>
</li>
</ul>
</div>
</div>

<div id="outline-container-sec-1" class="outline-2">
<h2 id="sec-1"><span class="section-number-2">1</span> Copy new ATAC-seq data from Luca/Pique-Regi lab to the rice cluster</h2>
<div class="outline-text-2" id="text-1">
<p>
Create directories in the rice cluster to save ATAC-seq data:
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/shared_data/internal_restricted/
mkdir atacseq_Luca_PiqueRegi
mkdir atacseq_Luca_PiqueRegi/rawbamfiles
mkdir atacseq_Luca_PiqueRegi/info
</pre>
</div>

<p>
Copy infomation on each file:
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/
scp gb9571@grid.wayne.edu:/nfs/rprscratch/atac.lcl/deseq/cv.txt info/
</pre>
</div>

<p>
Copy bam files:
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/
scp gb9571@grid.wayne.edu:/nfs/rprscratch/atac.lcl/merge/*.bam rawbamfiles/
</pre>
</div>

<p>
Copy merged bam files:
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/
scp gb9571@grid.wayne.edu:/nfs/rprscratch/atac.lcl/mergeReps/*.bam rawbamfiles/
</pre>
</div>
</div>
</div>

<div id="outline-container-sec-2" class="outline-2">
<h2 id="sec-2"><span class="section-number-2">2</span> Convert bam file to hdf5 format</h2>
<div class="outline-text-2" id="text-2">
</div><div id="outline-container-sec-2-1" class="outline-3">
<h3 id="sec-2-1"><span class="section-number-3">2.1</span> Prepare directory and install genome library, samtools, and python</h3>
<div class="outline-text-3" id="text-2-1">
</div><div id="outline-container-sec-2-1-1" class="outline-4">
<h4 id="sec-2-1-1"><span class="section-number-4">2.1.1</span> Create directory</h4>
<div class="outline-text-4" id="text-2-1-1">
<p>
Create directory to save hdf5 files:
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/shared_data/internal/
mkdir hdf5
cd hdf5
mkdir hg19
</pre>
</div>
</div>
</div>
<div id="outline-container-sec-2-1-2" class="outline-4">
<h4 id="sec-2-1-2"><span class="section-number-4">2.1.2</span> Install genome library</h4>
<div class="outline-text-4" id="text-2-1-2">
<p>
Clone Graham's genome repo and follow his instruction.
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/hjshim/software/genome
</pre>
</div>
</div>
</div>
<div id="outline-container-sec-2-1-3" class="outline-4">
<h4 id="sec-2-1-3"><span class="section-number-4">2.1.3</span> Install samtools</h4>
<div class="outline-text-4" id="text-2-1-3">
<p>
Download samtools on rice cluster:
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/software/source/
wget https://github.com/samtools/samtools/releases/download/1.3.1/samtools-1.3.1.tar.bz2
tar -vxjf samtools-1.3.1.tar.bz2
cd samtools-1.3.1
make
</pre>
</div>

<p>
Install it on rice cluster:
</p>
<div class="org-src-container">

<pre class="src src-bash">make prefix=~/software/install/samtools-1.3.1 install
</pre>
</div>

<p>
Make a symbolic link:
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/bin
ln -s ~/software/install/samtools-1.3.1/bin/* .
ls -al
</pre>
</div>
</div>
</div>

<div id="outline-container-sec-2-1-4" class="outline-4">
<h4 id="sec-2-1-4"><span class="section-number-4">2.1.4</span> Install python</h4>
<div class="outline-text-4" id="text-2-1-4">
<div class="org-src-container">

<pre class="src src-bash">module add python
pip install pysam --user
</pre>
</div>
</div>
</div>
</div>

<div id="outline-container-sec-2-2" class="outline-3">
<h3 id="sec-2-2"><span class="section-number-3">2.2</span> Copy chromosome.h5 file to rice cluster</h3>
<div class="outline-text-3" id="text-2-2">
<p>
I will copy the in PPS cluster to rice cluster here:
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/shared_data/internal/hdf5/hg19/
</pre>
</div>

<p>
Now login into PPS cluster and copy chromosome.h5 file:
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /data/internal/genome_db/hg19/
scp chromosome.h5 hjshim@rice.rcac.purdue.edu:/home/hjshim/d/shared_data/internal/hdf5/hg19/
</pre>
</div>
</div>
</div>

<div id="outline-container-sec-2-3" class="outline-3">
<h3 id="sec-2-3"><span class="section-number-3">2.3</span> Filter reads with low alignment quality 10</h3>
<div class="outline-text-3" id="text-2-3">
</div><div id="outline-container-sec-2-3-1" class="outline-4">
<h4 id="sec-2-3-1"><span class="section-number-4">2.3.1</span> Create directory to save preprocessed bam files</h4>
<div class="outline-text-4" id="text-2-3-1">
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/
mkdir processedbamfiles
</pre>
</div>
</div>
</div>
<div id="outline-container-sec-2-3-2" class="outline-4">
<h4 id="sec-2-3-2"><span class="section-number-4">2.3.2</span> Filter reads with low alignment quality 10</h4>
<div class="outline-text-4" id="text-2-3-2">
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/rawbamfiles/

samtools view -b -q 10 AT2-N701-N501.bam &gt; ../processedbamfiles/AT2-N701-N501.qfiltered10.bam
samtools view -b -q 10 AT2-N701-N502.bam &gt; ../processedbamfiles/AT2-N701-N502.qfiltered10.bam
samtools view -b -q 10 AT2-N701-N503.bam &gt; ../processedbamfiles/AT2-N701-N503.qfiltered10.bam

samtools view -b -q 10 AT2-N702-N501.bam &gt; ../processedbamfiles/AT2-N702-N501.qfiltered10.bam
samtools view -b -q 10 AT2-N702-N502.bam &gt; ../processedbamfiles/AT2-N702-N502.qfiltered10.bam
samtools view -b -q 10 AT2-N702-N503.bam &gt; ../processedbamfiles/AT2-N702-N503.qfiltered10.bam

samtools view -b -q 10 AT2-N703-N501.bam &gt; ../processedbamfiles/AT2-N703-N501.qfiltered10.bam
samtools view -b -q 10 AT2-N703-N502.bam &gt; ../processedbamfiles/AT2-N703-N502.qfiltered10.bam
samtools view -b -q 10 AT2-N703-N503.bam &gt; ../processedbamfiles/AT2-N703-N503.qfiltered10.bam

samtools view -b -q 10 AT2-N704-N501.bam &gt; ../processedbamfiles/AT2-N704-N501.qfiltered10.bam
samtools view -b -q 10 AT2-N704-N502.bam &gt; ../processedbamfiles/AT2-N704-N502.qfiltered10.bam
samtools view -b -q 10 AT2-N704-N503.bam &gt; ../processedbamfiles/AT2-N704-N503.qfiltered10.bam

samtools view -b -q 10 AT2-N705-N501.bam &gt; ../processedbamfiles/AT2-N705-N501.qfiltered10.bam
samtools view -b -q 10 AT2-N705-N502.bam &gt; ../processedbamfiles/AT2-N705-N502.qfiltered10.bam
samtools view -b -q 10 AT2-N705-N503.bam &gt; ../processedbamfiles/AT2-N705-N503.qfiltered10.bam

samtools view -b -q 10 AT2-N706-N501.bam &gt; ../processedbamfiles/AT2-N706-N501.qfiltered10.bam
samtools view -b -q 10 AT2-N706-N502.bam &gt; ../processedbamfiles/AT2-N706-N502.qfiltered10.bam
samtools view -b -q 10 AT2-N706-N503.bam &gt; ../processedbamfiles/AT2-N706-N503.qfiltered10.bam
</pre>
</div>
</div>
</div>
</div>

<div id="outline-container-sec-2-4" class="outline-3">
<h3 id="sec-2-4"><span class="section-number-3">2.4</span> Index filtered bam files</h3>
<div class="outline-text-3" id="text-2-4">
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/

samtools index AT2-N701-N501.qfiltered10.bam
samtools index AT2-N701-N502.qfiltered10.bam
samtools index AT2-N701-N503.qfiltered10.bam

samtools index AT2-N702-N501.qfiltered10.bam
samtools index AT2-N702-N502.qfiltered10.bam
samtools index AT2-N702-N503.qfiltered10.bam

samtools index AT2-N703-N501.qfiltered10.bam
samtools index AT2-N703-N502.qfiltered10.bam
samtools index AT2-N703-N503.qfiltered10.bam

samtools index AT2-N704-N501.qfiltered10.bam
samtools index AT2-N704-N502.qfiltered10.bam
samtools index AT2-N704-N503.qfiltered10.bam

samtools index AT2-N705-N501.qfiltered10.bam
samtools index AT2-N705-N502.qfiltered10.bam
samtools index AT2-N705-N503.qfiltered10.bam

samtools index AT2-N706-N501.qfiltered10.bam
samtools index AT2-N706-N502.qfiltered10.bam
samtools index AT2-N706-N503.qfiltered10.bam
</pre>
</div>
</div>
</div>

<div id="outline-container-sec-2-5" class="outline-3">
<h3 id="sec-2-5"><span class="section-number-3">2.5</span> Convert filtered bam files into hdf5</h3>
<div class="outline-text-3" id="text-2-5">
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/hjshim/software/genome/python/script/db/

python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N501.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N701N501.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N701-N501.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N502.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N701N502.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N701-N502.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N503.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N701N503.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N701-N503.qfiltered10.bam

python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N501.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N702N501.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N702-N501.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N502.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N702N502.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N702-N502.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N503.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N702N503.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N702-N503.qfiltered10.bam

python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N501.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N703N501.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N703-N501.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N502.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N703N502.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N703-N502.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N503.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N703N503.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N703-N503.qfiltered10.bam

python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N501.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N704N501.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N704-N501.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N502.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N704N502.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N704-N502.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N503.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N704N503.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N704-N503.qfiltered10.bam

python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N501.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N705N501.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N705-N501.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N502.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N705N502.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N705-N502.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N503.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N705N503.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N705-N503.qfiltered10.bam

python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N501.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N706N501.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N706-N501.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N502.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N706N502.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N706-N502.qfiltered10.bam
python load_bam_5prime_ends.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N503.qfiltered10.fwd /atacseq_Luca_PiqueRegi/N706N503.qfiltered10.rev /home/hjshim/d/shared_data/internal_restricted/atacseq_Luca_PiqueRegi/processedbamfiles/AT2-N706-N503.qfiltered10.bam
</pre>
</div>
</div>
</div>

<div id="outline-container-sec-2-6" class="outline-3">
<h3 id="sec-2-6"><span class="section-number-3">2.6</span> Obtain library read depth</h3>
<div class="outline-text-3" id="text-2-6">
<p>
First set track stats using set_track_stats.py and get library read depth using get_track_stats.py. One downside might be that it computes library read depth over all chromosomes including sex chromosomes.  
</p>
<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/hjshim/software/genome/python/script/db/

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N501.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N502.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N503.qfiltered10.fwd

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N501.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N502.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N503.qfiltered10.rev

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N501.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N502.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N503.qfiltered10.fwd

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N501.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N502.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N503.qfiltered10.rev

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N501.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N502.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N503.qfiltered10.fwd

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N501.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N502.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N503.qfiltered10.rev

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N501.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N502.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N503.qfiltered10.fwd

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N501.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N502.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N503.qfiltered10.rev

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N501.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N502.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N503.qfiltered10.fwd

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N501.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N502.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N503.qfiltered10.rev

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N501.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N502.qfiltered10.fwd
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N503.qfiltered10.fwd

python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N501.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N502.qfiltered10.rev
python set_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N503.qfiltered10.rev
</pre>
</div>

<div class="org-src-container">

<pre class="src src-bash">cd /home/hjshim/d/hjshim/software/genome/python/script/db/

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N501.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N502.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N503.qfiltered10.fwd

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N501.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N502.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N701N503.qfiltered10.rev

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N501.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N502.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N503.qfiltered10.fwd

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N501.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N502.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N702N503.qfiltered10.rev

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N501.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N502.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N503.qfiltered10.fwd

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N501.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N502.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N703N503.qfiltered10.rev

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N501.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N502.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N503.qfiltered10.fwd

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N501.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N502.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N704N503.qfiltered10.rev

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N501.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N502.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N503.qfiltered10.fwd

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N501.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N502.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N705N503.qfiltered10.rev

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N501.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N502.qfiltered10.fwd
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N503.qfiltered10.fwd

python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N501.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N502.qfiltered10.rev
python get_track_stats.py --assembly hg19 /atacseq_Luca_PiqueRegi/N706N503.qfiltered10.rev
</pre>
</div>

<p>
Library read depths have been saved in:
</p>
<div class="org-src-container">

<pre class="src src-bash">/home/hjshim/d/shared_data/internal/hdf5/hg19/atacseq_Luca_PiqueRegi/library.depth.txt
</pre>
</div>
</div>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="author">Author: Heejung Shim</p>
<p class="date">Created: 2016-11-01 Tue 12:39</p>
<p class="creator"><a href="http://www.gnu.org/software/emacs/">Emacs</a> 24.5.1 (<a href="http://orgmode.org">Org</a> mode 8.2.10)</p>
<p class="validation"><a href="http://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
